### PR TITLE
Include more data for DailyMedication storage to textfile

### LINKED ISSUE #200: [PE-D][Tester D] Dealing with 0 argument
Right now entering keyword followed by 0 arguments will gives me the help message, same output as what `-h` gives, even when there should be mandatory arguments.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: fxe025/ped#11

### PR DESCRIPTION
**Pull request by T0nyLin**
# Checklist for feature implementation and modification
Closes nil
- [X] Added relevant milestone (under milestone tab on the right)
- Implementation
  - [ ] Sources (if any) credited under both relevant section(s) and Developer Guide (Acknowledgement)
  - [ ] JUnit Test(s)
- Documentation
  - [X] JavaDoc for classes and methods
  - Developer Guide
    - [ ] Design (Can add as a todo statement for later on)
    - [ ] Implementation (Can add as a todo statement for later on)
    - [ ] Relevant User Story present
    - [ ] Non-functional requirement(s) that the feature addresses (if any)
    - [ ] Glossary of the (technical) term(s) used (if any)
    - [ ] Specialised instructions for manual testing (if any)
  - User Guide
    - [ ] Description and usage of feature (if any)
  - Contributions (under `/docs/team/*.md`)
    - [ ] Code/Feature/Enhancement contribution
    - [ ] Developer Guide contribution
    - [ ] User Guide contribution
    - [ ] Misc and team-based admin tasks (if any)

# Checklist for non-implementation related PR
Closes #200
- [X] Added relevant milestone (under milestone tab on the right)
- Documentation (Contributions)
  - [ ] Developer Guide contributions (if any)
  - [X] User Guide contributions (if any)
  - [ ] Review contributions (if any)
  - [ ] Miscellaneous and team-based admin tasks (if any)

# Additional Information

Nil

### CODE REVIEW

File: src/main/java/meditracker/dailymedication/DailyMedicationManager.java [hunk 1]
```diff
@@ -71,34 +61,20 @@ public static void importDailyMedicationManager(List<String> lines) {
...
-            addDailyMedication(dailyMedication, Period.MORNING);
-        } else if (period.equals("A")) {
-            addDailyMedication(dailyMedication, Period.AFTERNOON);
-        } else if (period.equals("E")) {
-            addDailyMedication(dailyMedication, Period.EVENING);
-        } else {
-            MEDILOGGER.warning("Assigned medication period not recognised. \"" +
-                    dailyMedication.getName() + "\" not imported into today's list.");
-        }
+    private static void addImportToSubLists(DailyMedication dailyMedication) {
```
Comment (wenenhoe) [inline src/main/java/meditracker/dailymedication/DailyMedicationManager.java:76]: This method appears to just be a wrapper for `addDailyMedication`. Might be better to just replace all usage of this method with `addDailyMedication` instead.

**Review by wenenhoe — CHANGES_REQUESTED** [review]
Generally LGTM! Good job on refactoring this to include more information for DailyMedication. Just a minor change required.

File: src/main/java/meditracker/dailymedication/DailyMedicationManager.java (hunk 1 above)
Comment (T0nyLin) [reply src/main/java/meditracker/dailymedication/DailyMedicationManager.java]: Implemented in 1fb50b4

**Review by wenenhoe — APPROVED** [review]
LGTM! Good job on including more data for DailyMedication and writing tests for ListCommand

**Pull request merged**